### PR TITLE
feat(instrumentation): Import redis from OpenTelemetry, add redis sample rag application

### PR DESCRIPTION
**Pull request by mpozniak95**
- [ ] If adding a new instrumentation or changing an existing one, I've added screenshots from some observability platform showing the change.
- [X] PR name follows conventional commits format: `feat(instrumentation): ...` or `fix(instrumentation): ...`.
- [ ] (If applicable) I have updated the documentation accordingly.

### CODE REVIEW

**Review by nirga — APPROVED** [review]
Thanks @mpozniak95! Mind rebasing the PR?

File: packages/sample-app/sample_app/redis_rag_app.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+from traceloop.sdk import Traceloop
+from traceloop.sdk.decorators import workflow
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+
+INDEX_NAME = "index"
+DOC_PREFIX = "doc:"
+EMBEDDINGS_DIM = 1536
+
+Traceloop.init(exporter=ConsoleSpanExporter())
+# Traceloop.init(app_name="redis_rag_app")
```
Comment (nirga) [inline packages/sample-app/sample_app/redis_rag_app.py:19]: @mpozniak95 maybe keep this line instead of the one above?

Comment (mpozniak95) [reply packages/sample-app/sample_app/redis_rag_app.py]: Done

**Comment by mpozniak95** [comment]
And new unused import :)

**Pull request merged**